packages/desktop-client/src/components/sort.tsx

```diff
@@ -51,8 +55,8 @@ export function useDraggable({
     },
     collect: monitor => ({ isDragging: monitor.isDragging() }),
 
-    end(item) {
-      _onDragChange.current({ state: 'end', type, item });
+    end(dragState) {
+      _onDragChange.current({ state: 'end', type, item: dragState.item });
```

Comment:
I've changed this - it was a bug - the type checking picked it up. 

The end function passes in the drag state as its prop. We were passing the full thing through as the "item" which should only contain the item that is being dragged. 

I have checked and this actually isn't getting used anywhere - so the bug wasn't visible anywhere.